apps/aecore/src/aecore_app.erl

```diff
@@ -8,7 +8,7 @@
          prep_stop/1,
          stop/1]).
 -export([check_env/0]).
-
+-export([set_level/1]).
```

Comment:
A note: this was actually only exported for manual testing, but could possibly be useful otherwise. I had intended to remove the export.